TWT/apps/timathon/views/judge/judge.py

```diff
@@ -71,6 +72,7 @@ def get(self, request: WSGIRequest):
                 average=Avg("c1") + Avg("c2") + Avg("c3") + Avg("c4")
             )["average"]
 
-        context["submissions"] = submissions
+        context["submissions"] = sorted(
+            submissions, key=lambda x: len(x.vote_set.all()))[::-1]
```

Comment:
this will make it even slower

Reply:
try to sort it on line https://github.com/Tech-With-Tim/twtcodejam.net/pull/68/files#R44

Reply:
ok done. commit fa46c6f changed that in line 44 as you requested :mhm: